src/api/src/services/FlashingStrategyLocator/artefacts.ts

```diff
@@ -110,8 +110,8 @@ export const removeDirectoryContents = async (firmwaresPath: string) => {
     return;
   }
   const files = await listFiles(firmwaresPath);
-  if (files.length > 4) {
```

Comment:
Are we removing more files than 4 now? I think we should be mindful about deleting files.

There are many stories where software deletes way more files than intended.

For example:

https://github.com/MrMEEE/bumblebee-Old-and-abbandoned/issues/123

Reply:
flasher.pyz is placing the firmware files it is configuring into the --dir folder, which is our binary-targets folder, so there are more than 4 files that are now being deleted when we clear the firmware files.

![image](https://github.com/ExpressLRS/ExpressLRS-Configurator/assets/38869875/305390f9-f076-4640-bd52-71abdf9b17f1)

I can increase the check to 7 instead of 4, but that just feels like we will continue to increase the number as things change, but I also do not know of a better way to make sure we are not deleting more files than we should.

Reply:
Lets do 20 then?